pkg/providers/imagefamily/customscriptsbootstrap/provisionclientbootstrap.go

```diff
@@ -108,8 +112,7 @@ func (p ProvisionClientBootstrap) GetCustomDataAndCSE(ctx context.Context) (stri
 		// GpuInstanceProfile:      lo.ToPtr(models.GPUInstanceProfileUnspecified), // Unsupported as of now (MIG)
 		// WorkloadRuntime:         lo.ToPtr(models.WorkloadRuntimeUnspecified),    // Unsupported as of now (Kata)
 		ArtifactStreamingProfile: &models.ArtifactStreamingProfile{
-			// TODO: Revisit when adding support for Azure Linux v3 and Ubuntu 24.04, this may have to be conditional then
-			Enabled: lo.ToPtr(true),
+			Enabled: to.Ptr(enableArtifactStreaming),
```

Comment:
why change from `lo` to `to`?

Reply:
vibe coding?

Reply:
Fixed. And no, we just have a mix in this file, and I happened to look at the following lines ... (Not changing the rest in this PR).